base/compiler/types.jl

```diff
@@ -1,4 +1,14 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
+#
+# Functions to fix up:
+#   abstract_eval_statement_expr
```

Comment:
```suggestion
```
The update for `abstract_eval_statement_expr` seems to have been done correctly?